web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java

```diff
@@ -503,4 +523,84 @@ public String startBulkAddParticipantJob(MultipartFile file, String schema, Stri
 		});
 		return jobDetail.getUuid();
 	}
+	
+	@ApiOperation(value = "To get PDF version casebook for one specific participant",  notes = "only work for authorized users with the right acecss permission ")
+	@RequestMapping(value = "/pdf/print/{studyOid}/{studySubjectIdentifier}", method = RequestMethod.POST)
```

Comment:
If participantId is unique within a study, let's use this url: /studies/{studyOID}/participants/{participantId}/casebook